src/databases/Blueprint/CMakeLists.txt

```diff
@@ -53,6 +54,9 @@ ${VISIT_INCLUDE_DIR}/avt/Pipeline/Sources
 ${VISIT_INCLUDE_DIR}/avt/VisWindow/VisWindow
 ${VISIT_INCLUDE_DIR}/visit_vtk/full
 ${VISIT_INCLUDE_DIR}/visit_vtk/lightweight
+${VTKh_INCLUDE_DIRS}
+${VTKM_DIR}/include/vtkm-1.2
+${VTKM_DIR}/include/vtkm-1.2/vtkm/thirdparty/taotuple
```

Comment:
not sure if this is needed in this PR?

Reply:
I think this somehow got pulled in from develop

Reply:
hmm, thats strange -- it shouldn't show as a diff?

Reply:
Unknown. I most certainly did not add this.

Reply:
I removed these since we don't need them